world/Plugins/Docker/docker.lua

```diff
@@ -288,100 +287,19 @@ function DockerCommand(Split, Player)
 end
 
 
+function OnPlayerFoodLevelChange(Player, NewFoodLevel)
+	-- Don't allow the player to get hungry
+	return true, Player, NewFoodLevel
+end
 
-function HandleRequest_Docker(Request)
-	
-	content = "[dockerclient]"
-
-	if Request.PostParams["action"] ~= nil then
-
-		action = Request.PostParams["action"]
-
-		-- receiving informations about one container
-		
-		if action == "containerInfos"
-		then
-			LOG("EVENT - containerInfos")
-
-			name = Request.PostParams["name"]
-			imageRepo = Request.PostParams["imageRepo"]
-			imageTag = Request.PostParams["imageTag"]
-			id = Request.PostParams["id"]
-			running = Request.PostParams["running"]
-
-			-- LOG("containerInfos running: " .. running)
-
-			state = CONTAINER_STOPPED
-			if running == "true" then
-				state = CONTAINER_RUNNING
-			end
-
-			updateContainer(id,name,imageRepo,imageTag,state)
-		end
-
-		if action == "startContainer"
-		then
-			LOG("EVENT - startContainer")
-
-			name = Request.PostParams["name"]
-			imageRepo = Request.PostParams["imageRepo"]
-			imageTag = Request.PostParams["imageTag"]
-			id = Request.PostParams["id"]
-
-			updateContainer(id,name,imageRepo,imageTag,CONTAINER_RUNNING)
-		end
-
-		if action == "createContainer"
-		then
-			LOG("EVENT - createContainer")
-
-			name = Request.PostParams["name"]
-			imageRepo = Request.PostParams["imageRepo"]
-			imageTag = Request.PostParams["imageTag"]
-			id = Request.PostParams["id"]
-
-			updateContainer(id,name,imageRepo,imageTag,CONTAINER_CREATED)
-		end
-
-		if action == "stopContainer"
-		then
-			LOG("EVENT - stopContainer")
-
-			name = Request.PostParams["name"]
-			imageRepo = Request.PostParams["imageRepo"]
-			imageTag = Request.PostParams["imageTag"]
-			id = Request.PostParams["id"]
-
-			updateContainer(id,name,imageRepo,imageTag,CONTAINER_STOPPED)
-		end
-
-		if action == "destroyContainer"
-		then
-			LOG("EVENT - destroyContainer")
-			id = Request.PostParams["id"]
-			destroyContainer(id)
-		end
-
-		if action == "stats"
-		then
-			id = Request.PostParams["id"]
-			cpu = Request.PostParams["cpu"]
-			ram = Request.PostParams["ram"]
-
-			updateStats(id,ram,cpu)
-		end
-
-
-		content = content .. "{action:\"" .. action .. "\"}"
-
-	else
-		content = content .. "{error:\"action requested\"}"
-
+function OnTakeDamage(Receiver, TDI)
+	-- Don't allow the player to take falling damage
+	dt_fall = 3
+	if Receiver:GetClass() == 'cPlayer' and TDI.DamageType == dt_fall
```

Comment:
The `dtFall` constant should be available globally, you should use the constant rather than a magic number, because the constant's integral value may change in a future version of the server.

Ref.: http://apidocs.cuberite.org/Globals.html#DamageType